packer/scripts/build_stable_dev_box.rb

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'json'
+require 'tempfile'
+
+def ansible_variable_json(version_hash)
+  foreman_branch = version_hash['foreman'] == 'nightly' ? 'develop' : "#{version_hash['foreman']}-stable"
+  katello_branch = version_hash['katello'] == 'nightly' ? 'master' : "KATELLO-#{version_hash['katello']}"
+  ansible_options = {
+      'katello_repositories_version': version_hash['katello'],
+      'foreman_repositories_version': version_hash['foreman'],
+      'pulpcore_repositories_version': version_hash['pulpcore'],
+      'pulp_repositories_version': version_hash['pulp'],
+      'foreman_installer_options': ["--katello-devel-katello-scm-revision=#{katello_branch}", "--katello-devel-foreman-scm-revision=#{foreman_branch}"]
+
+  }
+  ansible_options.to_json
+end
+
+def find_version(version_string)
+  if version_string.nil?
+    $stderr.print("No katello version specified")
+    exit -1
+  end
+
+  version_file =  "#{__dir__}/../../vagrant/config/versions.yaml"
```

Comment:
will this file definitely be updated for the version by the time the release dev gets to releasing this box? If it's not a sure thing we can point this out in the docs.

Reply:
If that file is not updated, no release pipelines can run. So as long this happens after RC1, you're good.

Reply:
great! :+1: